rbs/rbs_common.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RBS_COMMON_H
+#define RBS_COMMON_H
+
+extern "C" {
+#include "include/rbs.h"
+}
+
+#include "core/LocOffsets.h"
+
+namespace sorbet::rbs {
+
+/**
+ * A single RBS type comment found on a method definition or accessor.
+ *
+ * RBS type comments are formatted as `#: () -> void` for methods or `#: Integer` for attributes.
+ */
+struct Comment {
+    core::LocOffsets loc;    // The location of the comment in the file
+    std::string_view string; // The type string (excluding the `#: ` prefix)
```

Comment:
Can you add a comment about who owns this view?  I assume it's `GlobalState`'s file table, but it's not immediately obvious to me.

Reply:
This a view on the `File.source` data, we collect it in `RBSSignaturesWalk::findRBSComments`.  I added a comment 👍